modules/dxp/apps/commerce/commerce-machine-learning-forecast-alert-service/src/main/java/com/liferay/commerce/machine/learning/forecast/alert/service/impl/CommerceMLForecastAlertEntryServiceImpl.java

```diff
@@ -171,7 +182,7 @@ private long[] _getUserCommerceAccountIds(long userId)
 				"_portletResourcePermission",
```

Comment:
@JanisZhang Please also migrate this resource permission to the OSGi reference way.